### PR TITLE
Add Go Dep to the master branch

### PR DESCRIPTION
**Pull request by Zalgo2462**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -320,6 +320,9 @@ __install() {
 	__install_mongodb & __load "[+] Installing MongoDB"
 
 	( # Build RITA
+		# Ensure go dep is installed
+		go get -u github.com/golang/dep/cmd/dep
```
Comment (ethack) [inline install.sh]: Dep's readme recommends installing from a versioned release for stability. We'd be versioning our version manager too which seems like a good idea. Thoughts on downloading it this way instead?

```
wget -nv -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64
chmod +x $GOPATH/bin/dep
```

Comment (Zalgo2462) [reply install.sh]: awesome idea

**Comment by Zalgo2462** [comment]
I need to fix the dockerfile in this pr before this goes through

File: install.sh (hunk 1 above)
Comment (SamuelCarroll) [reply install.sh]: Looks great to me too!

**Comment by Zalgo2462** [comment]
Dockerfile fixed and dep version pinned.

**Pull request merged**